src/output/icons.rs

```diff
@@ -48,22 +48,26 @@ pub fn iconify_style(style: Style) -> Style {
 lazy_static! {
     static ref MAP_BY_NAME: HashMap<&'static str, char> = {
         let mut m = HashMap::new();
-        m.insert(".Trash", '\u{f1f8}'); // 
         m.insert(".atom", '\u{e764}'); // 
-        m.insert(".bashprofile", '\u{e615}'); // 
+        m.insert(".bashprofile", '\u{f489}'); // 
         m.insert(".bashrc", '\u{f489}'); // 
         m.insert(".git", '\u{f1d3}'); // 
         m.insert(".gitattributes", '\u{f1d3}'); // 
         m.insert(".gitconfig", '\u{f1d3}'); // 
         m.insert(".github", '\u{f408}'); // 
         m.insert(".gitignore", '\u{f1d3}'); // 
         m.insert(".gitmodules", '\u{f1d3}'); // 
+        m.insert(".pam_environment", '\u{e615}'); // 
         m.insert(".rvm", '\u{e21e}'); // 
+        m.insert(".Trash", '\u{f1f8}'); // 
         m.insert(".vimrc", '\u{e62b}'); // 
         m.insert(".vscode", '\u{e70c}'); // 
+        m.insert(".xinitrc", '\u{f489}'); // 
+        m.insert(".xprofile", '\u{f489}'); // 
         m.insert(".zshrc", '\u{f489}'); // 
-        m.insert("Cargo.lock", '\u{e7a8}'); // 
+        m.insert("AUTHORS", '\u{f718}'); // 
         m.insert("bin", '\u{e5fc}'); // 
+        m.insert("Cargo.lock", '\u{e7a8}'); // 
```

Comment:
this seems to be already present on main (at least it shows that icon when I run with `--icons`)